### ISSUE TITLE
Search Broken

### ISSUE THREAD

**Issue by silverbenz** [raindropio/app#113]
Search appears to be broken in Raindrop.io at the moment.

I have tried via an Alfred Workflow, which would typically list all matching results to my search query, but currently just offers to send me to Google - which is the fallback if no matching results are found. I've also tried via logging into the raindrop.io web app (in Firefox) and anything I type into the search field results in an Error page, with the image of a cat inside a TV. Tried the same search in Safari with the same result.

![image](https://github.com/raindropio/app/assets/10793755/7c7e01c7-082b-4a08-8d91-b2f5da7901bf)

Search context was "All bookmarks" and I have confirmed that all things I searched for do still exist in my bookmarks when I go through all bookmarks manually.

Bookmarks that I have searched for today were definitely working yesterday.

If there's anything else I can provide which might help, please let me know.

**Comment by g180** [comment]
Having the same issue on app.raindrop.io and Android app.
I would say the issue might started appearing last 5-7 hours, before then I had no problem like this.
Hopefully be fixed soon.

Thank devs for this great service.

**Comment by exentrich** [comment]
Disaster happen with our search index. We hosted our search engine on @elastic cloud. They just deleted our entire account.

Do not worry, all data is safe. I just need to rebuild search index on a different hosting provider. But this require few days to complete unfortunately

**Comment by g180** [comment]
Got it @exentrich. No big deal. Thanks for quick response.

**Comment by pencilcheck** [comment]
> Disaster happen with our search index. We hosted our search engine on @elastic cloud. They just deleted our entire account.
> 
> Do not worry, all data is safe. I just need to rebuild search index on a different hosting provider. But this require few days to complete unfortunately

wow, that is douchebag of them. What happened? why do they delete search index account like this??

**Comment by lilskippyy** [comment]
@exentrich wow! I hope everything is back on track soon

**Comment by exentrich** [comment]
> Elasticsearch reserves the right, in its sole discretion, to deny, restrict, suspend, discontinue, or terminate your account, with or without prior notice or explanation, for any or no reason, without any liability to you
https://www.elastic.co/legal/terms-of-use

Warning to anyone planning to use Elastic Cloud: They can terminate your account without prior notice for any reason. This is what happened to our account.
Despite multiple emails, they ignored us. Only one team member on Twitter responded, stating that they blocked my account because they mistakenly thought I was from Russia. However, I am a citizen of Kazakhstan and conduct business here. I hope they will contact me to provide a further explanation.

**Comment by g180** [comment]
Yeah, it a lot seems it was just their accident, if that appears to be factual then I believe Elastic must cover whatever costs Raindrop had to spend on this, hopefully they simply admit it.

As long as Electric doesn't bring a vivid logic on whatever their member on X quoted "sanctions" or "regulatory restrictions"--by who though--, they now might be taking their time to make up deal and strategy to this.

Well not that I want things escalate though. My best wishes to Raindrop.io on this.

**Comment by AlbertoFabbri93** [comment]
I am still having problems when searching bookmarks from the browser url bar, is the search index not fully rebuilt yet?
